studio/data/database-triggers/database-trigger-update-transaction-mutation.ts

```diff
@@ -41,7 +41,7 @@ export async function updateDatabaseTrigger({
 }: DatabaseTriggerUpdateVariables) {
   const sql = getDatabaseTriggerUpdateSQL({ originalTrigger, updatedTrigger })
   const { result } = await executeSql({ projectRef, connectionString, sql })
-  return result
+  return updatedTrigger
```

Comment:
this a mistake?

Reply:
Hey @MildTomato  caan u check this once now

Reply:
Yep, i think this is a mistake.